File: Calculator.py
def addemup(num1, num2):
    return num1 + num2

def subtractem(num1, num2):
    return num1 - num2

def multiplyem(num1, num2):
    return num1 * num2

def divideem(num1, num2):
    if num2 == 0:
        return print("Divide by zero error")
    else:
        return num1 / num2

def exponentiate(num1, num2):
    return num1 ** num2

def modulo(num1, num2):
    return num1 % num2

print(addemup(4, 2)) #should be 6
print(subtractem(17, 21)) #should be -4
print(multiplyem(3, -4)) #should be -12
print(divideem(17, 4)) #should be 4.25
print(exponentiate(3, 3)) #should be 27
print(modulo(100, 3)) #should be 1

File: Calculator.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Main.java to edit this template
 */
package calculator;

/**
 *
 * @author paulg
 */
public class Calculator {

    public static int addemup(int num1, int num2){
        return num1 + num2;
    }
    
    public static int subtractem(int num1, int num2){
        return num1 - num2;
    }
    
    public static int multiplyem(int num1, int num2){
        return num1*num2;
    }
    
    public static double divideem(double num1, double num2){
        if(num2 == 0){
            System.out.println("Divide by zero error!");
        }else{
            return num1 / num2;
        }
        return 0;
    }
    
    public static int exponentiate(int num1, int num2){
        int result = num1;
        for(int i = 1; i < num2; i++){
             result *= num1;
        }
        return result;
    }
    
    public static int modulo(int num1, int num2){
        return num1 % num2;
    }
   
    public static void main(String[] args) {
        System.out.println(addemup(4, 5));
        System.out.println(subtractem(4, 5));
        System.out.println(multiplyem(4, 5));
        System.out.println(divideem(4, 5));
        System.out.println(exponentiate(4, 5));
        System.out.println(modulo(4, 5));        
    }
}


